app/helpers/application_helper.rb

```diff
@@ -26,6 +26,15 @@ def isActivePage(path)
     return request.fullpath() == path
   end
 
+  # This is not the most elegant solution. Bootstrap though seems to add a '-tab' suffix
+  # to the end of the query string param, so just strip it out when comparing
+  # ---------------------------------------------------------------------------
+  def isActiveTab(tab)
```

Comment:
The logic to determine what tab is active should be best handled on the JS side. Bootstrap is not adding the tab query param, someone of us has added tab into the url form path for each of the three forms (e.g. https://github.com/DMPRoadmap/roadmap/blob/CDL-MVP/app/views/devise/registrations/_personal_details.html.erb#L1)

FYI, boostrap tabs click handler preventDefault which means it does not add the active tab to the URL. Why don't you read this thread (https://stackoverflow.com/questions/12131273/twitter-bootstrap-tabs-url-doesnt-change), it might help us to implement it on the JS side

Reply:
Yeah, I was originally going to do a JS solution for this, but the linter creates an error of 'window is not defined'. 
Since this was the only situation (that I noticed) where we had tabs that needed this functionality it was easy to just add it in the erb file. The plan tabs are unique pages so don't have the issue